build.gradle.kts

```diff
@@ -35,16 +35,13 @@ buildscript {
 }
 
 plugins {
+    kotlin(module = "plugin.serialization") version Versions.kotlin
     id("com.github.ben-manes.versions") version Versions.gradleVersionsPlugin
     id("com.appmattus.markdown") version Versions.markdownLintPlugin
 }
 
 apply(from = "gradle/scripts/detekt.gradle.kts")
 
-tasks.register<Delete>("clean") {
```

Comment:
Why remove this? It is used to clear out detect and markdownlint reports amongst other things

Reply:
I'll add it back in :)